Automattic/wp-calypso
# Earn: relaunch payments splash page update

This PR re-implements the Payments splash page updates that were originally made in 43601. Unfortunately that patch overwrote other changes made around the same time, and it was reverted in 44201. 

Here's a screenshot of layout:
<img width="1069" alt="Screen Shot 2020-07-17 at 1 03 38 PM" src="https://user-images.githubusercontent.com/35781181/87812353-0370db00-c82e-11ea-8c52-09938a323245.png">


#### Testing instructions

* Checkout this branch (git checkout update/earn-relaunch-payments-splash-page-update)
* Start Calypso
* Switch to a site that is on a paid plan but that does have a Stripe connection)
* Load the Payments page (/earn/payments/SITESLUG
* Confirm that you see the layout above.
